### PR TITLE
Logger2

### PR DESCRIPTION
**Pull request by t3chguy**

### CODE REVIEW

File: src/github.com/t3chguy/matrix-static/job-room-memberinfo.go [hunk 1]
```diff
@@ -14,18 +14,15 @@
...
+import "github.com/t3chguy/matrix-static/mxclient"
 
 type RoomMemberNotFoundError struct {
 	roomID string
 	mxid   string
 }
 
 func (err *RoomMemberNotFoundError) Error() string {
-	return fmt.Sprintf("Member %s not found in %s.", err.mxid, err.roomID)
+	return "Member " + err.mxid + " not found in " + err.roomID + "."
```
Comment (Kegsay) [inline src/github.com/t3chguy/matrix-static/job-room-memberinfo.go]: Why the change?

File: src/github.com/t3chguy/matrix-static/job-room-initialsync.go [hunk 2]
```diff
@@ -26,6 +28,7 @@ func (job RoomInitialSyncJob) Work(w *Worker) {
 	resp := &RoomInitialSyncResp{}
 
 	if _, exists := w.rooms[job.roomID]; !exists {
+		log.Infof("Worker [%d] - Initial Syncing Room [%s]", w.ID, job.roomID)
```
Comment (Kegsay) [inline src/github.com/t3chguy/matrix-static/job-room-initialsync.go]: It'd be nice to know when this finishes.

Comment (Kegsay) [inline src/github.com/t3chguy/matrix-static/job-room-initialsync.go]: What about back-pagination?

**Comment by Kegsay** [comment]
Otherwise LGTM

File: src/github.com/t3chguy/matrix-static/job-room-memberinfo.go (hunk 1 above)
Comment (t3chguy) [inline src/github.com/t3chguy/matrix-static/job-room-memberinfo.go]: I guess I was overly aggresive in replacing fmt everywhere

**Comment by t3chguy** [comment]
Its a disaster.
Everything is on fire.

**Comment by Kegsay** [comment]
LGTM
